### ISSUE TITLE
Menu Bar is missing when using "custom" Title Bar and nativeTabs setting is present

### ISSUE THREAD

**Issue by WilliamWsyHK** [Microsoft/vscode#60767]
My VSCode in Windows 10 x64 does not show menu bar `"window.menuBarVisibility": "default" // "toggle" or "visible" has no effect on this` when the setting of title bar is set to `"window.titleBarStyle": "custom"`.

The strange thing is in my other Windows machines (Windows 10 VM or WIndows 7 physical), those installs have no issue displaying menu bar (set to default); and this already appeared in several official stable builds before 1.28.1.

- VSCode Version: 1.28.1
- OS Version: Windows 10 x64 (10.0.17763.55; 1809)

Steps to Reproduce:

1. Open VSCode
2. Look for menu bar

Does this issue occur when all extensions are disabled?: Yes

settings.json
```json
{
    "editor.minimap.enabled": true,
    "editor.smoothScrolling": true,
    "terminal.integrated.shell.windows": "C:\\Windows\\SysNative\\WindowsPowerShell\\v1.0\\powershell.exe",
    "sync.gist": "...", // eradicated
    "sync.lastUpload": "...", // eradicated
    "sync.autoDownload": false,
    "sync.autoUpload": false,
    "sync.lastDownload": "...", // eradicated
    "sync.forceDownload": false,
    "sync.anonymousGist": false,
    "vsicons.dontShowNewVersionMessage": true,
    "workbench.iconTheme": "vscode-icons",
    "vsicons.projectDetection.autoReload": true,
    "sync.host": "",
    "sync.pathPrefix": "",
    "window.nativeTabs": true,
    "sync.quietSync": false,
    "sync.askGistName": false,
    "sync.removeExtensions": true,
    "sync.syncExtensions": true,
    "breadcrumbs.enabled": true
}
```

Screenshot
![image](https://user-images.githubusercontent.com/10591975/46864614-7a08c980-ce4d-11e8-9e8d-269bafa465d5.png)

**Comment by sbatten** [comment]
@WilliamWsyHK You are also missing the titlebar it seems?

**Comment by WilliamWsyHK** [comment]
Title bar is missing, but is expected, since the title bar setting is `"window.titleBarStyle": "custom"` by default. Did I missed something?

To make us clear, let me circle the part that I raise this issue.
![image](https://user-images.githubusercontent.com/10591975/46899453-1969a300-cec6-11e8-8991-780715270c20.png)
The whole menu bar is missing, even the "Minimize, Maximize, Close" trio.

Alright, I think I found the problem. Since I also have macOS with VSCode installed, and I have set `"window.nativeTabs": true`. This setting made Windows VSCode unable to display the menu bar (in the first comment https://github.com/Microsoft/vscode/issues/60767#issue-369496132).
